development/gcp/cloud-run/scan-logs-for-secrets/scan-logs-for-secrets.go

```diff
@@ -63,14 +63,17 @@ func main() {
 		mainLogger.LogCritical("failed to create storageClient: %s", err)
 	}
 
-	defer storageClient.Close()
-
 	// Setup configuration for gitleaks
-	viper.SetConfigType("toml")
-	if err := viper.ReadConfig(strings.NewReader(config.DefaultConfig)); err != nil {
-		mainLogger.LogError("failed read default viper config for leaks scanner, error: %s", err)
+	viper.SetConfigFile("app/development/gcp/cloud-run/scan-logs-for-secrets/gitleaks.toml")
```

Comment:
This file path can be simplified with a fixed Dockerfile

Reply:
Oh, I see. The same code is being executed twice. I will correct this.